tests/components/smartthings/test_scene.py

```diff
@@ -37,7 +37,7 @@ async def test_scene_activate(hass, scene):
     assert state.attributes["icon"] == scene.icon
     assert state.attributes["color"] == scene.color
     assert state.attributes["location_id"] == scene.location_id
-    # pylint: disable=protected-access
+    # pylint: disable-next=protected-access
     assert scene.execute.call_count == 1  # type: ignore
```

Comment:
Not sure this would actually raise `protected-access`. Haven't checked though.

Reply:
This one may be removed probably. We don't run pylint actively on our test suite.
Will check.

Reply:
Obsolete, removed